add BCM 2024 workshop (#278)
NeurodataWithoutBorders/nwb_hackathons: Workshop_2024_BCM/README.md

<img alt="banner" src="images/nwb_dandi_bcm_2024_banner.png">

# NWB and DANDI Workshop
## hosted by the Baylor College of Medicine Neuroscience Department

**Dates:** April 15-16, 2024

Join us for an interactive workshop on [Neurodata Without Borders (NWB)](https://nwb.org), a data standard for neurophysiology, and the [DANDI Archive](https://dandiarchive.org), a collaboration platform for publishing, sharing, and processing neurophysiology data. Learn how to use these resources to maximize the impact of your data while complying with the new NIH data sharing and management policies.

Representatives from the NWB and DANDI teams, Ryan Ly and Ben Dichter, will provide an overview of the NWB software ecosystem and teach you how to convert data to NWB and publish data on DANDI.

## Agenda

**April 15: NWB and DANDI training**
* 11-12: Introduction to NWB and DANDI
  * Core Format
  * Extensions
  * Analysis and visualization tools
  * Other NWB Events
* 12-1: Networking lunch
* 1-2: Converting data to NWB
  * NWB GUIDE
  * NeuroConv
  * PyNWB and MatNWB APIs
  * Extensions

> * 2-3: Publishing data the DANDI Archive

Reviewer: ```suggestion
* 2-3: Publishing data on the DANDI Archive
```

> * 2-3: Publishing data on the DANDI Archive

Reviewer: ```suggestion
* 2-3: Publishing and accessing data on the DANDI Archive
```

  * Uploading data
  * DANDI Metadata
  * Neurosift
  * DANDI Hub
  * Dendro
  * DANDI API

> **April 16, 9-3: NWB and DANDI hacking & office hours.** Participant will bring their own data and work with the NWB and DANDI teams to convert their data to NWB and publish it on DANDI.

Reviewer: ```suggestion
**April 16, 9am - 3pm: NWB and DANDI hacking & office hours.** Participant will bring their own data and work with the NWB and DANDI teams to convert their data to NWB and publish it on DANDI.
```
